docs/pages/guides/tutorials/Hello-Kubernetes-with-Dapr.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: default
+title: Hello Kubernetes with Dapr
+abstract: |
+  Dive into this step-by-step tutorial on deploying Dapr in Kubernetes using Meshery Playground. Learn to deploy a Python app for generating messages and a Node app for consuming and persisting them, all orchestrated with Dapr for enhanced application integration.
+permalink: guides/tutorials/hello-kubernetes-with-dapr
+redirect_from: guides/tutorials/Hello-Kubernetes-with-Dapr
+model: kubernetes
+type: guides
+category: tutorials
+language: en
+list: include
+---
+
+### Introduction :
+This tutorial shows how to deploy Dapr in a Kubernetes cluster using Meshery Playground. You'll deploy a Python app for message generation and a Node app for consumption and persistence, following the classic Hello World example with Dapr. Let's dive in and explore the architecture diagram that illustrates this setup.
+ ![](./screenshots/Architecture_Diag.png)
+
+### Prerequisites:
+- Basic understanding of containerization , Kubernetes concepts and basic knowledge of Dapr .
```

Comment:
remove extraneous whitespace before period.

What basic knowledge of Dapr do they need? Is this not potentially the very first time someone might touch Dapr?